all-the-icons.el

```diff
@@ -88,6 +88,7 @@
 (require 'cl-lib)
 
 (require 'data-alltheicons  "./data/data-alltheicons.el")
+(require 'data-devicons     "./data/data-devicons.el")
```

Comment:
devicons is already a part of file-icons, can you remove this extra font and its referneces, and update file-icons ttf its data file instead?

Reply:
Hello, I've checked https://github.com/file-icons/source/blob/master/charmap.md but found specific icon e.g. F# is not in `file-icons`. I don't know if I missed something.